src/zc/recipe/filestorage/tests.py

```diff
@@ -12,11 +12,12 @@
 #
 ##############################################################################
 
-import os, re, unittest
-import pkg_resources
```

Comment:
I guess this was removed intentional. It is not used directly, but I am not sure it used some magic or not, but I think its ok to remove it.